change/@fluentui-priority-overflow-09409455-f841-4490-8f29-242e315804df.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I think you can omit this change file as correct one is with minor bump and feature prefix